src/main/kotlin/com/openlattice/search/renderers/BHRAlertEmailRenderer.kt

```diff
@@ -28,6 +28,8 @@ private val INCIDENT_DATE_TIME_FQN = FullQualifiedName("bhr.datetimeOccurred")
 private val NATURE_OF_CRISIS_FQN = FullQualifiedName("bhr.dispatchReason")
 private val HOUSING_SITUATION_FQN = FullQualifiedName("housing.living_arrangements")
 private val DISPOSITION_FQN = FullQualifiedName("bhr.disposition")
+private val AFFILIATION_FQN = FullQualifiedName("bhr.affiliation")
+private val MILITARY_STATUS_FQN = FullQualifiedName("bhr.militaryStatus")
```

Comment:
Are these part of the v2 data model?

Reply:
No. Subscriptions are not available for v2 orgs. Will have to set up a new renderer and update the data model accordingly as only first degree neighbors are available to create the email content.